src/components/sbb-link/sbb-link.tsx

```diff
@@ -63,88 +76,134 @@ export class SbbLink {
   @Prop() public textSize: InterfaceLinkAttributes['textSize'] = 's';
 
   /**
-   * Choose the link style variant.
+   * Applies link inline styles (underline, inherit coloring/font-size etc).
    */
-  @Prop() public variant: InterfaceLinkAttributes['variant'] = 'block';
+  @Prop() public inline = false;
 
-  private get _inlineVariant(): boolean {
-    return this.variant === 'inline' || this.variant === 'inline-negative';
-  }
+  /**
+   * Disabled attribute if link is used as button (optional)
+   */
+  @Prop() public disabled?: boolean;
+
+  /**
+   * Name attribute if link is used as button (optional)
+   */
+  @Prop() public name?: string;
+
+  /**
+   * Form attribute if link is used as button (optional)
+   */
+  @Prop() public form?: string;
+
+  /**
+   * Type attribute if link is used as button (optional)
+   */
+  @Prop() public type?: InterfaceLinkAttributes['buttonType'];
 
+  /**
+   * Render element
+   */
   public render(): JSX.Element {
-    const textSizeClass = this._inlineVariant ? '' : ` sbb-link--text-${this.textSize}`;
+    let TAG_NAME;
+    if (this.isStatic) {
+      TAG_NAME = 'span';
+    } else if (this.href) {
+      TAG_NAME = 'a';
+    } else {
+      TAG_NAME = 'button';
+    }
+
+    return (
+      <TAG_NAME {...this._getAttributeList}>
+        {this.icon && !this.inline ? (
+          <span class="sbb-link__icon">
+            <slot name="icon" />
+          </span>
+        ) : (
+          ''
+        )}
+        <slot />
+      </TAG_NAME>
+    );
+  }
+
+  /**
+   * Get the attributelist base on the config and the resulting element
+   * @private
+   * @return <object>
+   */
+  private get _getAttributeList(): object {
+    let ariaLabelText = this.accessibilityLabel;
     const currentLanguage = getDocumentLang();
     const currentWritingMode = getDocumentWritingMode();
+    const attributeList: Record<string, string> = {};
+
+    Object.assign(attributeList, {
+      dir: currentWritingMode || undefined,
```

Comment:
If you are certain that the value cannot evaluate to falsy, you can remove the `|| undefined`